Lib/test/support/__init__.py

```diff
@@ -1181,8 +1181,9 @@ def requires_limited_api(test):
     return test
 
 
-TEST_MODULES_ENABLED = sysconfig.get_config_var('TEST_MODULES') == 'yes'
-
+# Windows build doesn't support --disable-test-modules feature, so there's no
+# 'TEST_MODULES' var in config
+TEST_MODULES_ENABLED = sys.platform == "win32" or sysconfig.get_config_var('TEST_MODULES') == 'yes'
```

Comment:
Can we perhaps interpret a missing value as 'yes'?

```suggestion
TEST_MODULES_ENABLED = (sysconfig.get_config_var('TEST_MODULES') or 'yes') == 'yes'
```

I'd rather the condition be about the configuration, rather than implicitly "knowing" that certain platforms have certain build options. (And I believe there are people who use MinGW make to build for `win32`, which would make the condition incorrect for them. We don't _support_ that, but no reason to make it harder on them.)

Reply:
This makes sense, thanks.